e2e/components.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { expect, test } from '@grafana/plugin-e2e';
+import { ExplorePage } from './fixtures/explore';
+
+test.describe('components', () => {
+  let explorePage: ExplorePage;
+  
+  test.beforeEach(async ({ page }) => {
+    explorePage = new ExplorePage(page);
```

Comment:
Why not using [Playwright fixtures](https://playwright.dev/docs/test-fixtures)?

Reply:
Feels cleaner to define the url once in the ExplorePage class and call the method. Is there another benefit to using Playwright fixtures here?

Reply:
Yes, by extending the [base test](https://playwright.dev/docs/test-fixtures#with-fixtures) (sorry I didn't link to the paragraph I had in mind), each test can received the `explorePage` as a param of the callback. For instance, like in [this suite of tests](https://github.com/grafana/explore-profiles/blob/main/e2e/tests/ad-hoc-view/ad-hoc-view.spec.ts).

You can pretty much [pass anything](https://github.com/grafana/explore-profiles/blob/main/e2e/fixtures/index.ts) that can help writing tests in a simpler way.